misc/tremulous-grangerhub.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Tremulous Grangerhub
+Comment=Aliens vs Humans, First Person Shooter game with elements of Real Time Strategy (Grangerhub version)
+Icon=tremulous-grangerhub
```

Comment:
We can change the icon file name to io.github.grangerhub.Tremulous.png instead. Same with the desktop file name (io.github.grangerhub.Tremulous.desktop).